### PR TITLE
CSP-60 added readme details

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,165 +1,39 @@
...
+# ![crest](https://assets.publishing.service.gov.uk/government/assets/crests/org_crest_27px-916806dcf065e7273830577de490d5c7c42f36ddec83e907efe62086785f24fb.png) Digital Apprenticeships Service
 
-# _Project Name_
+##  RoATP Course Management UI
 
 <img src="https://avatars.githubusercontent.com/u/9841374?s=200&v=4" align="right" alt="UK Government logo">
 
-_Update these badges with the correct information for this project. These give the status of the project at a glance and also sign-post developers to the appropriate resources they will need to get up and running_
 
-[![Build Status](https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_apis/build/status/_projectname_?branchName=master)](https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_build/latest?definitionId=_projectid_&branchName=master)
```
Comment (DevenKShah) [inline README.md:9]: Need these badges

File: README.md [hunk 2]
```diff
@@ -1,165 +1,39 @@
...
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=_projectId_&metric=alert_status)](https://sonarcloud.io/dashboard?id=_projectId_)
-[![Jira Project](https://img.shields.io/badge/Jira-Project-blue)](https://skillsfundingagency.atlassian.net/secure/RapidBoard.jspa?rapidView=564&projectKey=_projectKey_)
-[![Confluence Project](https://img.shields.io/badge/Confluence-Project-blue)](https://skillsfundingagency.atlassian.net/wiki/spaces/_pageurl_)
-[![License](https://img.shields.io/badge/license-MIT-lightgrey.svg?longCache=true&style=flat-square)](https://en.wikipedia.org/wiki/MIT_License)
+## About
 
-_Add a description of the project and the high-level features that it provides. This should give new developers an understanding of the background of the project and the reason for its existence._
+The front end for main providers to view and administer the courses they offer, and the venues or subregions where the courses are offered, and the provider description.
```
Comment (DevenKShah) [inline README.md:10]: Why `main providers`, this can be misinterpreted.

File: README.md [hunk 3]
```diff
@@ -1,165 +1,39 @@
...
 
-## How It Works
+#### Requirements
+- Clone this repository
+- Install [Visual Studio 2022](https://www.visualstudio.com/downloads/) with these workloads:
+    - ASP.NET and web development
+    - Azure development
+- Install [Azure Storage Emulator]
+- Install [Azure Storage Explorer](http://storageexplorer.com/)
+- Administrator Access
```
Comment (DevenKShah) [inline README.md:24]: Don't think you need admin access anymore

**Review by DevenKShah — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (MarkFCain) [reply README.md:10]: It's not for employer providers or supporting providers. I'll take out the word 'main'

File: README.md (hunk 3 above)
Comment (MarkFCain) [reply README.md:24]: ok

File: README.md (hunk 1 above)
Comment (MarkFCain) [reply README.md:9]: added

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**